mbzirc_ign/src/mbzirc_ign/bridges.py

```diff
@@ -102,3 +102,41 @@ def thrust_joint_pos(model_name, side):
         ign_type='ignition.msgs.Double',
         ros_type='std_msgs/msg/Float64',
         direction=BridgeDirection.ROS_TO_IGN)
+
+
+def arm_joint_states(world_name, model_name):
+    arm_prefix = f'/world/{world_name}/model/{model_name}/model/arm'
+    return Bridge(
+        ign_topic=f'{arm_prefix}/joint_state',
+        ros_topic='/joint_states',
+        ign_type='ignition.msgs.Model',
+        ros_type='sensor_msgs/msg/JointState',
+        direction=BridgeDirection.IGN_TO_ROS)
+
+
+def gripper_joint_states(world_name, model_name):
+    gripper_prefix = f'/world/{world_name}/model/{model_name}/model/arm/model/gripper'
+    return Bridge(
+        ign_topic=f'{gripper_prefix}/joint_state',
+        ros_topic='/joint_states',
```

Comment:
Should this topic be the same as the one used for the arm joint states?

Reply:
I've seen both cases before:

* `/joint_states`
* `/<namespace>/joint_states`

I think other packages like moveit subscribes to `/joint_states` by default but can be configured to subscribe to namespaced topics.

I changed to the form `/<namespace>/joint_states` for the arm and gripper in [00ed7c1](https://github.com/osrf/mbzirc/pull/72/commits/00ed7c1068bc0ab086710c6bddc70313dc8ae4b6). It's now:

`/usv/arm/joint_states`
`/usv/arm/gripper/joint_states`

The user can remap them back to `/joint_states` if they wish.